File: core/src/main/java/com/illuzionzstudios/customfishing/command/CustomFishingCommand.java
package com.illuzionzstudios.customfishing.command;

import com.illuzionzstudios.customfishing.command.sub.RewardsCommand;
import com.illuzionzstudios.mist.command.SpigotCommandGroup;
import com.illuzionzstudios.mist.command.type.ReloadCommand;

/**
 * Main command
 */
public class CustomFishingCommand extends SpigotCommandGroup {

    @Override
    public void registerSubcommands() {
        registerSubCommand(new ReloadCommand());
        registerSubCommand(new RewardsCommand());
    }

}

File: src/main/kotlin/com/illuzionzstudios/customfishing/command/CustomFishingCommand.kt
package com.illuzionzstudios.customfishing.command

import com.illuzionzstudios.mist.command.SpigotCommandGroup
import com.illuzionzstudios.mist.command.SpigotSubCommand
import com.illuzionzstudios.mist.command.response.ReturnType
import com.illuzionzstudios.mist.command.type.ReloadCommand

class CustomFishingCommand : SpigotCommandGroup() {

    override fun registerSubcommands() {
        registerSubCommand(ReloadCommand())
    }
}